src/Microsoft.DotNet.ProjectModel/CompilationOutputFiles.cs

```diff
@@ -58,14 +59,27 @@ public string PdbPath
 
         public virtual IEnumerable<string> Resources()
         {
-            var resourceNames = Project.Files.ResourceFiles
-                .Select(f => ResourceUtility.GetResourceCultureName(f.Key))
-                .Where(f => !string.IsNullOrEmpty(f))
-                .Distinct();
+            IEnumerable<string> resourceCultureNames = null;
+            var compilationOptions = Project.GetCompilerOptions(Framework, Configuration);
+            if (compilationOptions.EmbedInclude == null)
```

Comment:
That wouldn't really work because `Project.Files.ResourceFiles` returns `IEnumerable<string>` whereas `resolver.GetIncludeFiles("/")` returns `IEnumerable<IncludeEntry>`. But I agree the code isn't very clean, I'll try to clean this up somehow.